x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -1494,6 +1499,32 @@ export class RulesClient {
       );
     });
 
+    // update schedules only if schedule operation is present
+    const scheduleOperation = options.operations.find(
+      (op): op is Extract<BulkEditOperation, { field: Extract<BulkEditFields, 'schedule'> }> =>
+        op.field === 'schedule'
+    );
```

Comment:
what other operations should affect rescheduling of tasks?

Should `notifyWhen` or `throttle` trigger this bulkUpdateSchedules as well?